lib/basedriver/commands/actions.js

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash';
+import log from '../logger';
+import { errors } from '../../mjsonwp/errors';
+
+
+let commands = {}, helpers = {}, extensions = {};
+
+/**
+ * If w3c performActions was called, attempt to translate it to JSONWP actions, if possible.
+ */
+commands.performActions = async function (actions) {
+  let multiActions;
+  log.debug(`Attempting to translate W3C actions to JSONWP actions`);
+  multiActions = helpers.translateActions(actions);
```

Comment:
do we really need to do the translation? there is the separate /actions endpoint now. So let's implement the W3C protocol for it in each client lib and keep the old endpoints for JSONWP format. There are too many possible different chain variations, that we physically cannot cover with tests.

Reply:
If you implement it in a driver, this command will be overridden. I just figure it's better than nothing. What do you think @imurchie?

Reply:
my point is that it will be necessary for us to support one more abstraction layer. I'd rather keep it simple and let Appium just pass the request body to the server as is. So then the server provider could fix it on his side if there are issues.